ec2-agents.tf

```diff
@@ -1,149 +0,0 @@
-## Identity for Jenkins Controller infra.ci.jenkins.io
-data "aws_iam_user" "jenkins_infra_ci" {
-  user_name = "jenkins-infra-ci"
-}
```

Comment:
Note: to be deleted manually (tracked in https://github.com/jenkins-infra/helpdesk/issues/3662#issuecomment-1770472278)